tests/acceptance/features/apiSharingNg/sharedByMe.feature

```diff
@@ -1533,4 +1533,81 @@ Feature: resources shared by user
         }
       }
     }
-    """
\ No newline at end of file
+    """
+
+
+  Scenario Outline: user updates role of a share
+    Given user "Alice" has uploaded file with content "to share" to "/textfile1.txt"
+    And user "Alice" has created folder "FolderToShare"
+    And user "Alice" has sent the following share invitation:
+      | resource        | <path>             |
+      | space           | Personal           |
+      | sharee          | Brian              |
+      | shareType       | user               |
+      | permissionsRole | <permissions-role> |
+    When user "Alice" updates the last share with the following using the Graph API:
+      | updatedRole  | <updated-role>  |
```

Comment:
```suggestion
      | role  | <role>  |
```

Reply:
`role`, `permissionsRole` and `updatedRole` are confusing. what are they? :sweat_smile: